### DISCUSSION TITLE
Question: any wiki/example about using this container for self-hosting?

### DISCUSSION THREAD

**Discussion by romu70** [nginx-proxy/nginx-proxy#1980] [Q&A]
Hi,
First of all, I'm far from being a Docker expert. nginx-proxy seems the perfect solution to implement a reverse proxy for self-hosting (server behind a end user rooter). Just want to know if there is a wiki or such use case documented somewhere?

If not, I'll certainly be back with more specific questions. Thanks.

**Comment by tkw1536** [comment]
A basic example (using docker-compose) can be found in [the README](https://github.com/nginx-proxy/nginx-proxy#docker-compose).
